src/funk/fd_funk_val.h

```diff
@@ -40,8 +39,8 @@ fd_funk_val_max( fd_funk_rec_t const * rec ) { /* Assumes pointer in caller's ad
    NULL and vice versa.  Assumes no concurrent operations on rec. */
 
 FD_FN_PURE static inline void *         /* Lifetime is the lesser of rec or the value size is modified */
-fd_funk_val( fd_funk_rec_t *   rec,     /* Assumes pointer in caller's address space to a live funk record */
-             fd_wksp_t const * wksp ) { /* ==fd_funk_wksp( funk ) where funk is a current local join */
+fd_funk_val( fd_funk_rec_t const * rec,     /* Assumes pointer in caller's address space to a live funk record */
```

Comment:
This probably should return a `void const *` if it takes a `fd_funk_rec_t const *`.  Or maybe should have two versions for const correctness purposes.

Reply:
there are 2 versions. see fd_funk_val_const